src/hooks/useConnectedProvidersLite.js

```diff
@@ -23,13 +22,12 @@ import useStoreSettingsSelector from "./useStoreSettingsSelector";
  */
 const useConnectedProvidersLite = (internalId, type, onlyConnected, shouldExecute = true) => {
```

Comment:
Can you take this chance to rename it to useConnectedProvidersList? That's the correct name right?

Reply:
done